nrf_security/src/drivers/zephyr/Kconfig

```diff
@@ -0,0 +1,14 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+config PSA_CRYPTO_DRIVER_ENTROPY_ZEPHYR
```

Comment:
I think that we need to be careful with this configuration and make sure that ENTROPY_PSA_CRYPTO_RNG is not enabled at the same time. Since zephyr has an RNG driver which is using the PSA crypto APIs for RNG in a bad configuration this can be a cyclic dependency I think.

Reply:
That is true, we are going both directions with the API adapter layers here, but this is actually not anything new.
On devices that don't have nrf_cc3xx_platform support we are doing the same, however there it is mbedtls_hardware_poll that calls into the zephyr entropy API.

The configuration should be the same, one is depends on CRYPTOCELL_USABLE, the other !CRYPTOCELL_USABLE.
However in this implementation I've done the inverse of the PSA_CRYPTO_DRIVER_GENERATE_RANDOM_CC3XX_PLATFORM, which has depends on CRYPTOCELL_USABLE.
This should result in the same configuration.